Sources/PromotedCore/SchemaProtos/proto_common_common.pb.swift

```diff
@@ -601,19 +608,25 @@ extension Common_Properties: SwiftProtobuf.Message, SwiftProtobuf._MessageImplem
       // enabled. https://github.com/apple/swift-protobuf/issues/1034
       switch fieldNumber {
       case 1: try {
-        if self.structField != nil {try decoder.handleConflictingOneOf()}
         var v: Data?
         try decoder.decodeSingularBytesField(value: &v)
-        if let v = v {self.structField = .structBytes(v)}
+        if let v = v {
```

Comment:
are all these pb.swift files hand coded and not auto generated?

Reply:
They are autogenerated.